components/__tests__/AssetAddress.spec.ts

```diff
@@ -70,4 +70,17 @@ describe('AssetAddress', () => {
 
     expect(component.html()).toMatchSnapshot()
   })
+
+  it('should render with contract link', async () => {
```

Comment:
Should we test if the link not being rendered when `showContractLink` is `false` or `undefined`?

Reply:
It's false by default so it's covered by other tests.